apps/server/src/shared/repo/user/user.repo.ts

```diff
@@ -29,10 +29,7 @@ export class UserRepo extends BaseRepo<User> {
 			const { systems } = user.school;
 			return systems && systems.getItems().find((system) => system.id === systemId);
 		});
-		if (resultUser) {
-			return resultUser;
-		}
-		throw new NotFoundException('No user for this ldapId found');
+		return resultUser ?? Promise.reject();

```

Comment:
this seems like an unnecessary shorthand. Whats wrong with
`if (resultUser) return resultUser`
`Promise.reject()`?

Reply:
The  `??` operator is an often used shorthand for null checks in TS. It's use here is justified imo